api/tests/integration/infrastructure/repositories/user-repository_test.js

```diff
@@ -1119,20 +1119,31 @@ describe('Integration | Infrastructure | Repository | UserRepository', function
       });
 
       context('when user has authentication methods', function () {
-        it('should return the user with his authentication methods', async function () {
+        it('returns the user with his authentication methods', async function () {
```

Comment:
Suggestion : si c'est un nouveau standard, je suggère de 
- mettre à jour notre document sur les standards d'équipe dans le wiki
- s'assurer que toute l'équipe a bien pris en compte ce nouveau standard (le rappeler au daily)
- et faire la modif dans tous le fichier de test pour que ça soit cohérent.

Et niveau pix, ça serait cool d'en parler aussi sinon on va se retrouver avec des fichiers de tests avec du should et d'autres non > weekly engineering

My 2cts